pkg/istio-agent/agent_test.go

```diff
@@ -259,6 +261,47 @@ func TestAgent(t *testing.T) {
 			return a
 		}).Check(t, security.WorkloadKeyCertResourceName, security.RootCertReqResourceName)
 	})
+	t.Run("External SDS socket", func(t *testing.T) {
+
+		dir := mktemp()
+		copyCerts(t, dir)
+
+		secOpts := &security.Options{}
+		secOpts.RootCertFilePath = dir + "/root-cert.pem"
+		secOpts.CertChainFilePath = dir + "/cert-chain.pem"
+		secOpts.KeyFilePath = dir + "/key.pem"
```

Comment:
nit: shouldn't we use `filepath.Join(dir, "xxxx")` to keep the same conventions used in the rest of the tests?